### PR TITLE
Taskflow status

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
@kotfic This is ready from review

File: cumulus/common/girder.py [hunk 1]
```diff
@@ -69,6 +69,15 @@ def create_status_notifications(resource_name, notification, resource):
             })
 
 
+def send_status_notification(type, resource):
```
Comment (kotfic) [inline cumulus/common/girder.py]: nitpicky,  but can we not use ```type``` as the variable name?

Comment (cjh1) [inline cumulus/common/girder.py]: sure

**Comment by kotfic** [comment]
Is there a specific way you'd like me to test this?  If not from just reviewing the code everything looks good to me (other than nitpicks)

**Comment by cjh1** [comment]
Running taskflow_runner.py would be good.

**Comment by kotfic** [comment]
LGTM

**Pull request merged**